frontend/src/lib/components/heroEmoji/HeroEmoji.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import type {HeroEmojiProps} from './HeroEmoji.type';
+
+  type $$Props = HeroEmojiProps;
```

Comment:
This is not being used

Reply:
True, but it is needed to associate `HeroEmojiProps` as the component's allowed properties. The effect is not huge, though.